modules/v2x/v2x_proxy/os_interface/os_interface.h

```diff
@@ -21,108 +21,86 @@
 
 #pragma once
 
+#include <memory.h>
```

Comment:
Lol. Do you used `memcpy` -like operations. `<memory.h>` is seldom used. Prefer `<cstring>` if possible.